background.js

```diff
@@ -1,23 +1,31 @@
-chrome.pageAction.onClicked.addListener(function(tab) {
-  chrome.tabs.executeScript(null, { file: "src/startContentProcessor.js" });
-});
+requirejs.config(requirejsConfig);
```

Comment:
It's much more normal to include this call in the config file itself I think. Makes it clearer how that other file is used, and stops this file depending on extra globals existing.